docs/src/recipes/common-patterns.md

```diff
@@ -40,18 +76,24 @@ the token so that it is only usable for that specific request.
 Then if the request's token gets stolen, it will limit its impact.
 
 Let's use a basic token containing a user id, that would have access to everything
-owned by that user, and do a PUT HTTP request on "/articles/1":
+owned by that user, and do a GET HTTP request on "/articles/1":
 
 <bc-datalog-playground showBlocks="true">
   <code class="block">
 user(1234);
   </code>
   <code class="authorizer">
-// the authorizer provides the current date, the API endpoint and HTTP method
+// the authorizer provides the current date, the resource being accessed and the operation being performed
```

Comment:
Small detail but here you list date, resource and operation but below it is listed in a different order (date, operation, resource). If possible, I would put "resource("/articles/1") before operation("read") :)

Reply:
i noticed it as well and thought nobody else would noticed or be bothered by it.

haha